tools/sfxconvert.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+
+import os
+import argparse
+
+AudioFunctions = {}
+Verbose = False
+Verbose2 = False
+
+def set_verbose(v):
+    global Verbose
+    global Verbose2
+
+    Verbose = v
+    Verbose2 = v
+    return 1
+
+def make_audio_dict(AudioFunctions, repo):
+    with open(repo + os.sep + 'include' + os.sep + 'functions.h','r') as funcfile:
+        funcdata = funcfile.readlines()
+    for i, line in enumerate(funcdata):
+        if(line.count('sfxId')):
+            funcname, argnum = get_func_data(funcdata,i)
+            if(funcname != None):
+                AudioFunctions[funcname] = argnum
+    return 1
+
+def get_func_data(funcdata,i):
+    j = i
+    while(funcdata[j - 1].count(';') == 0):
+        j -= 1
+    k = i + 1
+    while(funcdata[k - 1].count(';') == 0):
+        k += 1
```

Comment:
I know you're just porting this and probably didn't write yourself, but man thats hard to read

Reply:
Yeah, I peeked the code and I went "Nope"...